src/app/forms/components/defect-select/defect-select.component.scss

```diff
@@ -86,3 +90,33 @@
     }
   }
 }
+
+li .ref {
```

Comment:
styling on this component doesn't seem to quite match the service design, the last child of the select component seems to match the styling, however the root and first child column fields don't seem to match